app/views/admin/tags/_tag.html.haml

```diff
@@ -0,0 +1,28 @@
+.batch-table__row{ class: [!tag.requires_review? && !tag.usable? && 'batch-table__row--muted'] }
+  .batch-table__row__content.batch-table__row__content--padded.pending-account
+    .pending-account__header
```

Comment:
I really don't like this relying on the pending-account UI, since it's just generic batch-table contents stuff by now, but I'm also not thinking that refactoring that would at all be in scope of this PR.